snap/validate.go

```diff
@@ -47,6 +57,15 @@ func Validate(info *Info) error {
 		return err
 	}
 
+	epoch := info.Epoch
+	if epoch == "" {
+		return fmt.Errorf("snap epoch cannot be empty")
```

Comment:
> My vote would be to fix the store to ensure it does not produce invalid infos, but either way would be fine with me.

Alright, is [this](https://github.com/ubuntu-core/snappy/pull/1189/commits/06a303fb6681386136f60bbbb914bb9cd43646e6) what you have in mind? It'll make sure we have a default until the store starts sending them.